content_scripts/VirtualCamera.js

```diff
@@ -733,6 +870,18 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
         return [A[0] + B[0], A[1] + B[1], A[2] + B[2]];
     }
 
+    // specifically used to add velocity to position but unscale position to keep the same distance from position to focus cube position
+    // A - position/targetPosition, v - velocity to add, F - focus cube position
```

Comment:
There's a small typo and the documentation would be a bit more standard as the following:
```javascript
/**
 * specifically used to add velocity to position but unscale position to keep the same distance from position to focus cube position
 * @param {Array<number>} P - position/targetPosition
 * @param {Array<number>} v - velocity to add
 * @param {Array<number>} F - focus cube position
 */
```